src/lava/magma/compiler/channels/pypychannel.py

```diff
@@ -230,7 +235,10 @@ def _req_callback(self):
         try:
             while not self._done:
                 self._req.recv_bytes(0)
+                not_empty = self.probe()
```

Comment:
This loop is still busy waiting, isn't it? Is this a performance concern? Could this also be improved in performance by using a condition?

Reply:
There no busy waiting here, because self._req.recv_bytes() would have blocked.